alby.go

```diff
@@ -329,6 +330,79 @@ func (svc *AlbyOAuthService) GetBalance(ctx context.Context, senderPubkey string
 	return 0, errors.New(errorPayload.Message)
 }
 
+func (svc *AlbyOAuthService) ListInvoices(ctx context.Context, senderPubkey, from, until, limit, offset string) (invoices []*Invoice, err error) {
```

Comment:
from, until, limit, offset should not be strings, right? (same with LND)

Reply:
Fixed